README.md

```diff
@@ -158,39 +166,31 @@ qvm-tags sd-dev add sd-client
+
+Then run the following command to set up a development environment:
 
 ```
 make dev
 ```
 
 Note that this target automatically sets the `environment` variable in `config.json` to `dev`, regardless of its current value, before provisioning.
 
-The build process takes quite a while. You will be presented with a dialog asking how to connect to Tor: you should be able to select the default option and continue. You may wish to increase the scrollback in the dom0 terminal from 1000 (the default) to 100000, to ensure you can review any errors in the verbose output. If you want to refer back to the provisioning log for a given
+The build process takes quite a while. You will be presented with a dialog asking how to connect to Tor: you should be able to select the default option and continue. If you want to refer back to the provisioning log for a given
 VM, go to `/var/log/qubes/mgmt-<vm name>.log` in `dom0`. You can also monitor logs as they're being written via `journalctl -ef`. This will display logs across the entire system so it can be noisy. It's best used when you know what to look for, at least somewhat, or if you're provisioning one VM at a time.
 
-**NOTE:** Due to [issue #202](https://github.com/freedomofpress/securedrop-workstation/issues/202), the installation may fail with disk quota errors. If this happens, reboot the entire workstation and run `make dev` again. The error will contain the following informating in your dom0 terminal:
-
-```
-qfile-agent : Fatal error: File copy: Disk quota exceeded; Last file: <...> (error type: Disk quota exceeded) '/usr/lib/qubes/qrexec-client-vm dom0 qubes.Receiveupdates /usr/lib/qubes/qfile-agent /var/lib/qubes/dom0-updates/packages/*.rpm' failed with exit code 1!
-```
-
 When the installation process completes, a number of new VMs will be available on your machine, all prefixed with `sd-`.
 
 #### Editing the configuration
 When developing on the Workstation, make sure to edit files in `sd-dev`, then copy them to dom0 via `make clone && make dev` to reinstall them. Any changes that you make to the ~/securedrop-workstation folder in dom0 will be overwritten during `make clone`. Similarly, any changes you make to e.g. `/srv/salt/` in dom0 will be overwritten by `make dev`.
 
 ### Staging Environment
 
-The staging environment differs from a production envionment in that it builds a local RPM, installs it in dom0, uses the dom0 package repository configuration for future updates of the RPM package from the https://yum-test.securedrop.org repository, and makes it so that you receive the latest nightlies of the workstation components, such as the SecureDrop Client.
```

Comment:
@eloquence this could be useful to add to the Development Environment section, what do you think?

Reply:
I think the removed text as written is inaccurate:
- The staging environment now uses `main` instead of `nightly` (this is noted as part of this PR).
- The staging environment doesn't _inherently_ build or install an RPM like this text suggests. You could install an RPM from yum-test and then run `sdw-admin --apply` with the environment set to `staging`, which this README also documents.

I think it might help to clarify a bit more when to use the `Makefile` targets to build and install the RPM, and when to manually install the RPM.

Reply:
oops, I meant the part about how the RPM is built locally and installed when you run `make dev`

Reply:
Tried to clarify this in a couple of places in d22b80c03878f3a00a726123c4427d95e830ceca